acc_estimation_main:
  hdf5_open_file_path: /home/ericlab/Downloads/acc_real.hdf5
  estimation_name: estimation
  object_detect_mode: ssd
  object_detect_checkpoint_path: /home/ericlab/tsuchida/2022_12/checkpoints/SSD/full_randomize/HV8/model_005000.pth
  semantic_class_num: 2
  semantic_checkpoint_path: /home/ericlab/tsuchida/2023_01/checkpoints/Semseg/HV8/sensor_b_box_2000_noize/latest.pth

decide_object_position:
  sensor_name: phoxi_camera
  sensor_angle_max: 0.087
  sensor_angle_min: 0.087
  distance: 1.1
  z_position: 0.003
  box_height: 0.01
  box_name: denso_box
  object_height: 0.15

data_transform_pkg:
  f_scale: 0.5
  cx_scale: 0.99
  cy_scale: 0.9